datasets/factckbr/factckbr.py

```diff
@@ -103,6 +120,11 @@ def _generate_examples(self, filepath):
                 if id_ == 0:
                     continue
 
+                label = row[8].lower()
+
+                if label == "":
+                    continue
```

Comment:
Are there rows with missing labels ?

In this case we should keep them and replace the missing label value with `-1`

The user is free to remove the bad rows using `.filter` for example

Reply:
There are 4 examples with missing labels, but I wasn't aware of this convention. Sorry!